products/amazon-corretto.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Amazon Corretto
+category: lang
+iconSlug: openjdk
+permalink: /amazon-corretto
+alternate_urls:
+-   /corretto
+versionCommand: java -version
+releasePolicyLink: https://aws.amazon.com/corretto/faqs/
+changelogTemplate: "https://github.com/corretto/corretto-{{'__LATEST__'|split:'.'|first}}/blob/release-__LATEST__/CHANGELOG.md"
+releaseDateColumn: true
+
+# There is one repository for each major release (except for 15 and 16).
+auto:
+-   git: https://github.com/corretto/corretto-jdk.git
+-   git: https://github.com/corretto/corretto-8.git
+-   git: https://github.com/corretto/corretto-11.git
+-   git: https://github.com/corretto/corretto-17.git
+-   git: https://github.com/corretto/corretto-18.git
+-   git: https://github.com/corretto/corretto-19.git
+
+# Do not forget to update the "auto" configuration on each new major release.
+# EOL dates can be found on https://aws.amazon.com/corretto/faqs/.
+releases:
+-   releaseCycle: "19"
+    eol: 2023-04-01
+    latest: "19.0.2.7.1"
+    releaseDate: 2022-09-20
+    latestReleaseDate: 2023-01-17
+
+-   releaseCycle: "18"
+    eol: 2022-10-18
+    latest: "18.0.2.9.1"
+    releaseDate: 2022-03-22
+    latestReleaseDate: 2022-07-19
+
+-   releaseCycle: "17"
+    lts: true
+    eol: 2029-10-01
+    latest: "17.0.6.10.1"
+    releaseDate: 2021-09-16
+    latestReleaseDate: 2023-01-17
+
+-   releaseCycle: "16"
+    eol: 2021-10-19
+    latest: "16.0.2.7.1"
+    releaseDate: 2021-03-16
+    latestReleaseDate: 2021-07-20
+    link: https://github.com/corretto/corretto-jdk/blob/release-16.0.2.7.1/CHANGELOG.md
```

Comment:
untested, but we should support this if we don't already.

```suggestion
    link: https://github.com/corretto/corretto-jdk/blob/release-__LATEST__/CHANGELOG.md
```

Reply:
It's not supported for the time being, opened #2449.

Reply:
Now that 2449 is merged it is now possible. Change applied.